pom.xml

```diff
@@ -193,14 +193,13 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.22.0</version>
+				<version>3.0.0-M1</version>
 				<configuration>
 					<!-- required due to a bug in the latest JDK - see https://stackoverflow.com/questions/50661648/spring-boot-fails-to-run-maven-surefire-plugin-classnotfoundexception-org-apache/50661649#50661649 -->
-	                <useSystemClassLoader>false</useSystemClassLoader>
+					<useSystemClassLoader>false</useSystemClassLoader>
```

Comment:
this is not necessary anymore with surefire 3.0.0 and should be removed since it can lead to problems if you indeed want to load a file from the classpath during mvn test.